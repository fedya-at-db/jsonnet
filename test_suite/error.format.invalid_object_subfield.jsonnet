"%(a.b.c)s" % {a: {c: "hello"}}

