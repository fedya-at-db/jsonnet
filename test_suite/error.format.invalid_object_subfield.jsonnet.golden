RUNTIME ERROR: No such partial field: a.b
	std.jsonnet:<stdlib_position_redacted>	function <get_nested_val>
	std.jsonnet:<stdlib_position_redacted>	function <get_nested_val>
	std.jsonnet:<stdlib_position_redacted>	thunk <val>
	std.jsonnet:<stdlib_position_redacted>	thunk <val>
	std.jsonnet:<stdlib_position_redacted>	thunk <a>
	std.jsonnet:<stdlib_position_redacted>	
	std.jsonnet:<stdlib_position_redacted>	thunk <a>
	std.jsonnet:<stdlib_position_redacted>	function <anonymous>
	std.jsonnet:<stdlib_position_redacted>	function <anonymous>
	std.jsonnet:<stdlib_position_redacted>	function <format_code>
	...
	std.jsonnet:<stdlib_position_redacted>	function <aux>
	std.jsonnet:<stdlib_position_redacted>	function <padding>
	std.jsonnet:<stdlib_position_redacted>	function <pad_left>
	std.jsonnet:<stdlib_position_redacted>	thunk <s_padded>
	std.jsonnet:<stdlib_position_redacted>	thunk <v>
	std.jsonnet:<stdlib_position_redacted>	function <format_codes_obj>
	std.jsonnet:<stdlib_position_redacted>	function <format_codes_obj>
	std.jsonnet:<stdlib_position_redacted>	function <anonymous>
	std.jsonnet:<stdlib_position_redacted>	function <anonymous>
	error.format.invalid_object_subfield.jsonnet:1:1-32	
